x-team/auto-ui
# Create ActivityLogsDisplay component

**Release Type:** *non-breaking feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-2551

## Description

Create ActivityLogsDisplay component

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-2551-update-activity-module-frontend` | [Auto](https://github.com/x-team/auto/pull/1693)

## Steps to Test or Reproduce

1. Pull latest from this branch and run `npm install` to install `faker`
2. Run `npm start` and reach out to http://localhost:9001/?selectedKind=UI%20Components%2FActivityLogsDisplay&selectedStory=complete
3. Observe that, in order to expose visual typography vulnerabilities, the data is generated randomly
    - If only `label` is present, display it with the light font-weight
    - If `label` and `value` is present, display the label in light font-weight and value with bold font-weight, separated by `:`
    - If only `value` is present, display it with the light font-weight 

## Impacted Areas in Application

- new component `ActivityLogsDisplay`

## Screenshots

![screen recording 2018-12-07 at 11 12 am](https://user-images.githubusercontent.com/131859/49649739-2790f700-fa11-11e8-92e1-c5baeac515a1.gif)
